scene/gui/popup_menu.cpp

```diff
@@ -710,9 +710,7 @@ void PopupMenu::_input_from_window_internal(const Ref<InputEvent> &p_event) {
 			search_string = "";
 		}
 
-		if (String::chr(k->get_unicode()) != search_string) {
-			search_string += String::chr(k->get_unicode());
-		}
+		search_string += String::chr(k->get_unicode());
```

Comment:
This prevents jumping between entries starting with the same letter (Windows explorer works like that for example). Not sure if that's improvement.